classes/Project/Project.mjs

```diff
@@ -0,0 +1,65 @@
+import dbDriver from "../../database/driver.mjs"
+import {validateProjectPayload} from "../../utilities/validatePayload.mjs"
+import {User} from "../User/User.mjs"
+import ProjectFactory from "./ProjectFactory.mjs"
+
+let err_out = Object.assign(new Error(), {
```

Comment:
Is this ever used as a 500? This might be simpler as just a `new Error()` or removes and declared when needed below.